docs/scos/dev/upgradability-services/upgradability-guidelines/upgradability-guidelines.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Evaluator reference
+description: Find solutions to Evaluator errors
+template: concept-topic-template
+---
+
+The documents in this section will help you resolve the issues related to code evaluation in a way that keeps your code upgradable and up to date with Spryker's and industry coding standards. 
+
+When you get an evaluation error, check the name of the triggered check in the Evaluation output logs. The name is at the beginning of each error log.
+
+Example:
+```bash
+-------------------- ----------------------------------------------------------------------------------------------------
+PrivateApi:Extension Please avoid extension of the PrivateApi Spryker\Zed\CustomerAccessGui\Communication\Form\CustomerAccessForm in Pyz\Zed\CustomerAccessGui\Communication\Form\CustomerAccessForm
+-------------------- ----------------------------------------------------------------------------------------------------
+```
+
+In the example, the name is `PrivateApi:Extension`. To find the documentation for this error, check the name in the table below.
+
+| Group name |  Group description  | Check name  | Error message template | Documentation |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| NotUnique | Covers the cases that are related to project-level entity names that can conflict with the entity names on the core level. | NotUnique:TransferName | Transfer object name **{transfer_name}** has to have project prefix Pyz in **{absolute_transfer_path}**, like **Pyz{transfer_name}** |  {link - unique-entity-name-not-unique.md#Making transfer names unique}  |
```

Comment:
In the error message template, the placeholders are in bold. Is that so in the code?

Reply:
I think it does not matter. 
Feel free to change it